openassessment/xblock/submission_mixin.py

```diff
@@ -671,6 +672,7 @@ def get_files_info_from_user_state(self, username):
                             download_url=download_url,
                             description=description,
                             name=file_name,
+                            size=None,
```

Comment:
Should we update the docstring to match?

Reply:
Yeah, this is actually super outdated